src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,67 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String[] FILE_STRING_VALUES = {"supply", "buy", "result"};
+    private static final String COMA = ",";
+    private static final int[] FILE_INT_VALUES = {0, 0, 0};
+    private static final int MINIMUM_FILE_LENGTH = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String fileText = readFromFile(fromFileName);
```

Comment:
when you have the text that you read as String - lets pass this string to method `createReport()` that returns ready report as a String

inside this method you will split your text to lines, then split each line by comma, and gather information inside 2 variables - `totalBuy` and `totalSupply`. then use StringBuilder to prepare the report